src/components/data/DataTable.js

```diff
@@ -120,8 +120,9 @@ const DataTable = props => {
     Utils.withBusyState(setLoading),
     withErrorReporting('Error loading entities')
   )(async () => {
-    const results = await Ajax(signal).Workspaces.workspace(namespace, name).entitiesOfType(entityType)
-    setSelected(entityMap(results))
+    const params = _.merge({ pageSize: filteredCount }, activeTextFilter ? { filterTerms: activeTextFilter } : {})
```

Comment:
is there a better way to do this? I never want to send an empty filterTerms query parameter, i.e. `?filterTerms=&pageSize=10` should never happen.